src/org/opendatakit/briefcase/ui/reused/ExportConfigurationButton.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.ui.reused;
+
+import static java.awt.Color.DARK_GRAY;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Insets;
+import javax.swing.JButton;
+
+
+/**
+ * Contains application logic pertaining to the Export Configuration Button
+ */
+public class ExportConfigurationButton extends JButton implements Comparable<ExportConfigurationButton> {
```

Comment:
The following may seem strange, but we always want to stay away from extending classes when we're not implementing Polymorphism, which is not the case in this instance, because we're not really trying to [replace conditional logic](https://refactoring.com/catalog/replaceConditionalWithPolymorphism.html).

I've gone ahead and prepared an alternative implementation of `ExportConfigurationButton` that wraps around a `Jbutton` instance and has the same behavior without extending `JButton`.  You can check it here: https://github.com/ggalmazor/briefcase/commit/e7bb0891403a54a13892f9d6439f753e9536d363